### ISSUE TITLE
Sync cloud data *down* from subsurface-mobile to desktop

### ISSUE THREAD

**Issue by MiM-TMassey** [Subsurface-divelog/subsurface#3473]
### Describe the issue:

- [ ] Bug
- [ ] Change request
- [X] New feature request
- [X] Discussion request

### Issue long description:
My Bluetooth dongle broke, so I can't download dives into Subsurface Windows.  No problem:  I can download them to Subsurface-Mobile, and sync with the cloud, right?  This works fine; but no matter what I try to do I can't get them to show up in Subsurface Windows.  The only thing I've achieved is to overwrite the cloud with the (older) data in Subsurface.

In short, I can completely overwrite the cloud data with (older) desktop data, but never *sync* desktop data with (newer) cloud data from subsurface-mobile.

### Operating system:
Windows 10
iOS 15
Shearwater Perdix

### Subsurface version:
Windows 5.0.8.
Latest version iOS app (2022/06/30)
Downloaded pre-compiled Windows version and latest app in app store.

### Steps to reproduce:
- PC:  
1. Start Subsurface and open log file.
2. File/Preferences/Cloud, type password, Save.  File/Save to Cloud Storage.
-  Web:
1. Log into cloud.subsurface-divelog.org/ and confirm that my cloud data exactly matches my desktop data.
- Phone:
1. Optional:  Delete subsurface-mobile app (to start from fresh config).
2. Use App store to download subsurface-mobile.
3. Start app and log into cloud account.  My data appears, matching desktop and web.
4. Go through process to download latest dives from computer:  new dives show up.
5. Hamburger menu/Manually sync with cloud
- Web:
1. <CTRL><F5>:  Reload the cloud data.  My new dives appear there.

Now, no matter what I do on the Subsurface side, I can't get it to synchronize with the cloud data.  If I File/Open cloud storage, I can see the new data.  But I can't figure out how to get that data into my *local* file.  And if I File/Save to cloud storage, it makes mention of some sort of sync, but I always end up with only my older desktop data completely overwriting the cloud data.

I'm not sure if I'm doing something wrong, if the documentation doesn't describe how to do it (or I've missed it), or if this is just a feature that isn't implemented!  :)

### Current behavior:
Desktop data can only overwrite cloud data:  it never syncs with new data from the cloud.

### Expected behavior:
Desktop data is synchronized with cloud data, allowing me to get my new dives that were uploaded to the cloud using subsurface-mobile.

### Additional information:
Happy to supply whatever might be requested.

**Comment by ghost** [comment]
This is from the perspective of a Mac user, the menues on Windows may be different.
I use my iPhone to get the dive data from my Shearwater and then sync the newly imported dive to the cloud service. Next I open Subsurface on my Mac and choose Menu -> File -> Open cloud storage. I edit the new dive as wanted and sync it back to the cloud storage (Menu -> File -> Save to cloud storage).
Then I do a manual sync on the iPhone.
Before I quit Subsurface on the Mac I save the data locally (Menu -> File -> Save as) and chose to overwrite the local file. The file location is set in the preferences of Subsurface under Dive log. Thus I have the same data locally on my Mac [1] and in the cloud.
At the same place you can choose which data file should be the default.

[1] Which is nice while being on a boat for a week without internet service.

**Comment by MiM-TMassey** [comment]
Sorry for the delay: I tested that process on Windows:  works fine.  Never would have tried that in a million years:  thank you for the suggestion.  It's definitely *not* a "sync" function, but it works.

I did it a *little* different, but the end result was the same:  I opened SS and opened my log file, then saved it to the cloud.  I deleted the phone app and reinstalled it, downloaded the dive data, downloaded from my computer and synced data with the cloud.  I then opened the cloud storage on my PC and immediately saved the data to a new file on my PC.  I then modified the dive data as I wanted, saved the data to cloud storage, and exited SS.  I then moved the old file to a backup folder and renamed the new file in its place.  I then manually synced with cloud on my phone.

Easy peasy!  :)  No, not really:  I wish the PC side would sync just like the mobile app would...  But it does work after a fashion:  my dives are off my dive computer and into my PC, which is what I wanted.  Thank you *very* much for your help!
